apps/jetbrains/jetbrains.py

```diff
@@ -298,7 +298,7 @@ def select_range(line_start: int, line_end: int):
         # if it's a single line, select the entire thing including the ending new-line5
         if line_start == line_end:
             actions.user.idea(f"goto {line_start} 0")
-            actions.user.idea("action EditorSelectLine"),
+            (actions.user.idea("action EditorSelectLine"),)
```

Comment:
Looks like this was a mistake that got auto formatted to make it more obvious — just remove the trailing comma.